### DISCUSSION TITLE
Penumbra body materials

### DISCUSSION THREAD

**Discussion by Uberfailz** [xivdev/Penumbra#164] [Q&A]
Is there a tutorial on how to change body materials in Penumbra? I don't see any options for it and I've been wondering how to.

**Comment by Ottermandias** [comment]
Enable Advanced Options, Enable Edit Mode, hover over 'convert' for help.

**Reply by Uberfailz** [reply]
![PIYdnJG](https://user-images.githubusercontent.com/101058302/156934588-af19d4d0-2794-4d86-8d78-d2fa135f8f90.png)
I don't see those options in my penumbra am I on the wrong version?

**Comment by Ottermandias** [comment]
When Advanced is on, you have Enable Editing in the Installed Mods Tab when selecting a mod.
![image](https://user-images.githubusercontent.com/70807659/156934690-0b836225-932b-4c6c-b357-0e068f1c95a6.png)

**Reply by Uberfailz** [reply]
Got it to work thank you very much!
